proto/src/determined/experiment/v1/searcher.proto

```diff
@@ -165,7 +165,12 @@ message CloseTrialOperation {
 // Shut down custom searcher method.
 message ShutDownOperation {
   // Cannot have an empty message type.
+  // Deprecated: But maintained here for backwards compatibility
   int32 placeholder = 1;
```

Comment:
It's an internal API, so I vote to just break it.  Also the comment above this line.

Reply:
i would agree to remove this.

Reply:
Makes sense, testing the changes to remove this